R/FilterStateRange.R

```diff
@@ -67,21 +69,25 @@ RangeFilterState <- R6::R6Class( # nolint
     },
 
     #' @description
-    #' Returns a formatted string representing this `LogicalFilterState`.
+    #' Returns a formatted string representing this `RangeFilterState`.
     #'
-    #' @param indent (`numeric(1)`) the number of spaces before after each new line character of the formatted string.
-    #' Default: 0
+    #' @param indent (`numeric(1)`)
+    #'        the number of spaces before after each new line character of the formatted string.
+    #'        Default: 0
     #' @return `character(1)` the formatted string
     #'
     format = function(indent = 0) {
       checkmate::assert_number(indent, finite = TRUE, lower = 0)
 
+      # vals <- pmax(pmin(self$get_selected(), private$choices[2]), private$choices[1])
```

Comment:
I think format should match condition?

![image](https://user-images.githubusercontent.com/15201933/217268630-954f6fd1-1c2b-47a4-91ad-2df74e44e48e.png)

Reply:
This should be discussed.
Do we want to print the range selected by the user, e.g. _AGE 30-39_, or the range of the actual data, e.g. _AGE 30-38_, because the user selected `30-39` but there are no patients older than 38 in the dataset?

I think the former (_30-39_) as that way different documents can be compared without confusion, although there is an argument to be made that the report should reflect the data and not the user's wishes.

Reply:
Having discussed with @nikolas-burkoff and @mhallal1 we decided that the format should reflect the selection (input state), not the data.